power-pay-backend/src/main/java/com/adorsys/gis/powerpay/powerpaybackend/repository/MoneyTransferValidateRepo.java

```diff
@@ -0,0 +1,11 @@
+package com.adorsys.gis.powerpay.powerpaybackend.repository;
+
+import com.adorsys.gis.powerpay.powerpaybackend.domain.Transaction;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public class MoneyTransferValidateRepo {
+    public interface MoneyTransferRepo extends JpaRepository<Transaction, String> {
+    }
+}
```

Comment:
This is a very wrong pattern in Spring Boot.

Reply:
Oh Yes sir, my bad it's a mistake.
my MoneyTransferRepo interface is nested inside the MoneyTransferValidateRepo class
which is a wrong approach as you said.